test/WebSites/ViewComponentWebSite/Startup.cs

```diff
@@ -15,7 +16,13 @@ public void Configure(IApplicationBuilder app)
             {
                 services.AddMvc(configuration);
             });
-            app.UseMvc();
+
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute(
+                    name: "default",
+                    template: "{controller=Home}/{action=Index}/{id?}");
```

Comment:
:shipit: 

And please open a bug for discussion about

> public static IRouteBuilder MapDefaultMvcRoute(this IRouteBuilder routeCollectionBuilder) extension?

My current thinking is that we don't want it and it will be covered by scaffolding the project template. The intent for this was to support easy start for trying things out, and not for demoware. The method should be more like `UseMvcWithDefaultRoute()` or something more short an sweet if we do it at all. Please CC @Eilon on the new bug.